services/core-web/src/components/mine/Projects/ProjectSummary.tsx

```diff
@@ -239,6 +242,8 @@ export const ProjectSummary: FC = () => {
         handleTabChange(newActiveTab);
         setIsLoaded(true);
       }
+
+      dispatch(fetchActivities(userInfo?.preferred_username));
```

Comment:
The only real change is having this dispatch here so when we use the `getActivities` selector in the NotificationDrawer component it will have the latest activity feed information.